Frontend/ui-common/src/Engineering/Helpers/JBridge.js

```diff
@@ -1933,9 +1933,16 @@ export const factoryResetApp = function (str) {
   }
 }
 
-export const uploadFile = function (unit) {
+export const uploadFile = function (aspectRatio) {
   return function () {
-    return JBridge.uploadFile();
+    try{
+      aspectRatio.showAccordingToAspectRatio = aspectRatio.showAccordingToAspectRatio.toString();
+      aspectRatio.imageAspectHeight = aspectRatio.imageAspectHeight.toString();
+      aspectRatio.imageAspectWidth = aspectRatio.imageAspectWidth.toString();
+      return JBridge.uploadFile(JSON.stringify(aspectRatio));
```

Comment:
@ROHIT-43 can we add a argument count check such that this becomes backward compatible